### PR TITLE
[DOC] CoreML frontend tutorial

### PR DESCRIPTION
**Pull request by kazum**
@zhreshold @zhiics, please help to review.

### CODE REVIEW

File: tutorials/frontend/from_coreml.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+######################################################################
+# Load pretrained CoreML model
+# ----------------------------
+# We will download and load a pretrained mobilenet classification network
+# provided by apple in this example
+model_url = 'https://docs-assets.developer.apple.com/coreml/models/MobileNet.mlmodel'
+model_file = 'mobilenet.mlmodel'
+download(model_url, model_file)
+# now you mobilenet.mlmodel on disk
```
Comment (zhiics) [inline tutorials/frontend/from_coreml.py]:
```suggestion
# Now you have mobilenet.mlmodel on disk
```

File: tutorials/frontend/from_coreml.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+x = np.transpose(img, (2, 0, 1))[np.newaxis, :]
+
+######################################################################
+# Compile the model on Relay
+# ---------------------------
+# We should be familiar with the process right now.
+target = 'cuda'
+shape_dict = {'image': x.shape}
+
+# parse CoreML model and convert into Relay computation graph
```
Comment (zhiics) [inline tutorials/frontend/from_coreml.py]:
```suggestion
# Parse CoreML model and convert into Relay computation graph
```

**Comment by zhiics** [comment]
LGTM.  @yzhliu Please take a look since you wrote the CoreML Relay frontend.

File: tutorials/frontend/from_coreml.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+# Compile the model on Relay
+# ---------------------------
+# We should be familiar with the process right now.
+target = 'cuda'
+shape_dict = {'image': x.shape}
+
+# Parse CoreML model and convert into Relay computation graph
+func, params = relay.frontend.from_coreml(mlmodel, shape_dict)
+
+with relay.build_config(opt_level=2, add_pass=['AlterOpLayout']):
```
Comment (yzhliu) [inline tutorials/frontend/from_coreml.py]: why not directly using `opt_level=3`?

**Comment by kazum** [comment]
@yzhliu, addressed your comment.  In addition, I added the original author @zhreshold to the author list.  Can you take a look again?

**Comment by yzhliu** [comment]
Thanks @kazum

**Pull request merged**